Source/WebCore/platform/graphics/FontCascade.h

```diff
@@ -151,12 +152,14 @@ class FontCascade : public CanMakeWeakPtr<FontCascade>, public CanMakeCheckedPtr
 
     bool isSmallCaps() const { return m_fontDescription.variantCaps() == FontVariantCaps::Small; }
 
-    float wordSpacing() const { return m_wordSpacing; }
-    float letterSpacing() const { return m_letterSpacing; }
+    float letterSpacing() const;
+    float wordSpacing() const;
+    const Length& computedLetterSpacing() const { return m_spacing.letter; }
+    const Length& computedWordSpacing() const { return m_spacing.word; }
+    void setLetterSpacing(const Length& s) { m_spacing.letter = s; }
```

Comment:
We generally don't use single letter parameter, maybe `length` ?

Reply:
Sure, I can change it to `spacing`. The original was using `s` (see below), so I just left it unchanged.